test/z-app.test.js

```diff
@@ -0,0 +1,391 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
```

Comment:
the file name is called `z-app.test.js`.  Is it possible to rename the file name to fit what it's supposed to do?

Reply:
@dhmlau run the `app.test.js` first causes other tests fail, I couldn't figure it out within one day so I prefer to leave it out of the scope of this PR :( I can create an issue to fix it afterwards.

Reply:
i see. so the tests in this file is only a temp place to know what tests we're skipping now? i'm good with a follow up PR to handle the rest.

Reply:
I am not happy about moving renaming this test file from `app.test.js` to `z-app.test.js` and back, because it prevents `git` from rendering a useful `diff` showing the changes made alongside the transfer.

If it's too difficult to fix these test properly, then I am proposing the following workaround:
1. In this pull request, keep the file name as `app.test.js` and mark the tests as skipped via `describe.skip` at top-level.
2. In the next pull request, rename the file to `z-app.test.js` and remove `.skip`.

Or maybe do both changes in the same pull request, but split them into two commits.

Thoughts?

Also BTW, the test name `z-app.test.js` does not make this test file the last one, there is still `test/zosconnectee.test.js` that's run after this one.

Reply:
@bajtos True the code compare is pretty bad :( skipping it makes sense to me 👌